test/box-luatest/trigger_yield_test.lua

```diff
@@ -0,0 +1,47 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function()
+    g.server = server:new({alias = 'trigger_yield'})
+    g.server:start()
+end)
+
+g.after_all(function()
+    g.server:stop()
+end)
+
+g.test_trigger_yield = function()
+    local n_tuples = g.server:exec(function()
+        local fiber = require('fiber')
+
+        local function fail()
+            fiber.sleep(0.0001)
+            error('fail')
+        end
+
+        local function insert()
+            box.space.test:auto_increment{fiber.id()}
+        end
+
+        box.schema.space.create('test')
+        box.space.test:create_index('pk')
+        box.space.test:truncate()
```

Comment:
Nit: Seems unnecessary, the space is just created.

Reply:
Fixed:

```diff
diff --git a/test/box-luatest/trigger_yield_test.lua b/test/box-luatest/trigger_yield_test.lua
index c0df397b55..1db0b9e6a4 100644
--- a/test/box-luatest/trigger_yield_test.lua
+++ b/test/box-luatest/trigger_yield_test.lua
@@ -27,7 +27,6 @@ g.test_trigger_yield = function()
 
         box.schema.space.create('test')
         box.space.test:create_index('pk')
-        box.space.test:truncate()
         box.space.test:on_replace(fail)
 
         local fibers = {}
```